internal/proto/mlserver/modelrepo/model_repository.proto

```diff
@@ -1,8 +1,9 @@
-// Copied from https://github.com/SeldonIO/MLServer/blob/9a7c77f93dd61ac395e389280195d8adc413aad7/proto/model_repository.proto
 syntax = "proto3";
 
 package inference.model_repository;
 
+// Copied from https://github.com/SeldonIO/MLServer/blob/9358dd2de7e309471c2382364289d56f000be6fe/proto/model_repository.proto
+
```

Comment:
This comment was moved below the package definition just so that it isn't copied into the generated .pb.go file